relay-server/src/actors/processor.rs

```diff
@@ -2318,7 +2318,12 @@ impl Service for EnvelopeProcessorService {
 mod tests {
     use chrono::{DateTime, TimeZone, Utc};
     use relay_general::pii::{DataScrubbingConfig, PiiConfig};
+    use relay_general::protocol::EventId;
+    use relay_sampling::{RuleCondition, RuleType, SamplingMode};
 
+    use crate::service::ServiceState;
+    use crate::testutils::{new_envelope, state_with_rule_and_condition};
+    use crate::utils::Semaphore as MySemaphore;
```

Comment:
Can we remove this to `TestSemaphore` instead? 
```suggestion
    use crate::utils::Semaphore as TestSemaphore;
```